connections.py

```diff
@@ -129,23 +131,27 @@ def handle_invalid_vehicle_id(self, vehicle_id):
                                                    arr_station)
                 if arr_reliability is not None:
                     connection['arrival']['reliability_graph'] = arr_reliability
+                    print(arr_reliability)
 
                 if 'vias' in connection:
                     for via in connection['vias']['via']:
                         via_station = via['stationinfo']['@id']
-                        dep_vehicle_id = self.handle_invalid_vehicle_id(via['departure']['vehicle'].split('.')[-1])
-                        arr_vehicle_id = self.handle_invalid_vehicle_id(via['arrival']['vehicle'].split('.')[-1])
+                        dep_vehicle_id = re.findall(r'\d+', via['departure']['vehicle'])[0]  #self.handle_invalid_vehicle_id(connection['departure']['vehicle'].split('.')[-1])
+                        arr_vehicle_id = re.findall(r'\d+', via['arrival']['vehicle'])[0] #self.handle_invalid_vehicle_id(connection['arrival']['vehicle'].split('.')[-1])
+                        print("\t{} {}".format(arr_vehicle_id, dep_vehicle_id))
```

Comment:
Excuse me sir, do you have a moment to talk about our lord and savior, the f-string?

```python
print(f'\t{arr_vehicle_id} {dep_vehicle_id}')
```